README.md

```diff
@@ -91,7 +113,17 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+
+function doSomething(thingtoDo){
+
+}
+
+doSomething(function(){
+
+});
+
```

Comment:
When called, `doSomething` needs to invoke `thingToDo`. We're looking for an answer along these lines...

```js
function doSomething(thingToDo){
  thingToDo();
}
doSomething(function(){
  alert("Yay!");
});
```